ewatercycle/models/abstract.py

```diff
@@ -68,6 +91,52 @@ def set_value(self, name: str, value: np.ndarray) -> None:
         """
         self.bmi.set_value(name, value)
 
+    def set_value_at_indices(self, name: str, indices: Iterable[int], value: np.ndarray) -> None:
+        """Specify a new value for a model variable at particular indices.
+
+        Args:
+            name: Name of variable
+            indices: The indices into the variable array
+            value: The new value for the specified variable.
```

Comment:
Currently, the [get_value](https://github.com/eWaterCycle/ewatercycle/blob/a7c8d3a2e9ff4ccfaa7f5cd1e503d3e8818c9ce5/ewatercycle/models/abstract.py#L52) returns a `np.ndarray`. I think this is used for 1D, too because `np.array` is not a type.